### PR TITLE
add changes to quiambao branch

### PR DESCRIPTION
**Pull request by quiambao-mapatriciaanne**
Please check and review my code.

### CODE REVIEW

File: bnpackage/quiambao.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+         print("1. Basic Info")
+         print("2. Goals")
+         print("3. Comments")
+         print("4. Exit")
+         choice = int(input("Enter your choice: "))
+
+        match choice:
+
+            case 1:
+                print(f"I'm 20 yearss old Now.")
```
Comment (ynion-mabeamae) [inline bnpackage/quiambao.py]: Please check your spelling.

**Review by ynion-mabeamae — CHANGES_REQUESTED** [review]
Please check your code before I approved.

File: bnpackage/quiambao.py (hunk 1 above)
Comment (delima-justine) [inline bnpackage/quiambao.py]: Remove the extra 's' in "yearss" and make the N lower case (now).

File: bnpackage/quiambao.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            case 2:
+                print(f"Continuosly acquire new skil. while furthur improving my coding skil.")
```
Comment (citron-kathleen) [inline bnpackage/quiambao.py]: Please check your code.

**Review by citron-kathleen — CHANGES_REQUESTED** [review]
Please check your spelling.

File: bnpackage/quiambao.py (hunk 2 above)
Comment (delima-justine) [inline bnpackage/quiambao.py]: Complete the word "skill", add an 'l' and make the w in while uppercase(W).

File: bnpackage/quiambao.py (hunk 1 above)
Comment (delima-justine) [inline bnpackage/quiambao.py]: Remove the f-string.

**Review by delima-justine — CHANGES_REQUESTED** [review]
Please review your code again based on the recommendations before I approve. Thank you!

File: bnpackage/quiambao.py (hunk 2 above)
Comment (relente-patriciajoy) [inline bnpackage/quiambao.py]: Please check the typos and remove f-string to your code.

**Review by relente-patriciajoy — CHANGES_REQUESTED** [review]
Please check your code before I approve it.

**Review by delima-justine — APPROVED** [review]
Great!

**Review by relente-patriciajoy — APPROVED** [review]
Good job!

**Review by citron-kathleen — APPROVED** [review]
Good job!

**Review by ynion-mabeamae — APPROVED** [review]
Good job!

**Pull request merged**